src/components/TrackHistory/index.js

```diff
@@ -31,4 +66,10 @@ const styles = StyleSheet.create({
   title: {fontSize: 12, textAlign: 'center'},
 });
 
-export default TrackHistory;
+function mapStateToProps(state) {
+  return {transactionHistory: state.transactions.transactionHistory};
+}
+
+const mapDispatchToProps = {changeModalView};
+
+export default connect(mapStateToProps, mapDispatchToProps)(TrackHistory);
```

Comment:
Use hook based apis like `useDispatch` & `useSelector` instead of `connect`

https://react-redux.js.org/api/hooks